Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+        lets_certificate=args.get('lets_certificate'),
+        multi_certificate=args.get('multi_certificate'),
+        certificate_group=args.get('certificate_group'),
+        certificate=args.get('certificate'),
+        intergroup=args.get('intergroup'),
+        ip_range=args.get('ip_range'),
+    )
+    command_results = generate_simple_command_results(object_type=OutputTitle.SERVER_POLICY.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.UPDATED.value,
+                                                      response=response)
+
+    return command_results
+
+
+def server_policy_delete_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Delete a server policy.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    name = args['name']
+    response = client.server_policy_delete_request(name)
+    command_results = generate_simple_command_results(object_type=OutputTitle.SERVER_POLICY.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.DELETED.value,
+                                                      response=response)
+
+    return command_results
+
+
+def server_policy_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """List server policies.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    name = args.get('name')
+    response = client.server_policy_list_request(name=name)
+    parsed_data, pagination_message, formatted_response = list_response_handler(
+        client=client,
+        response=response,
+        data_parser=client.parser.parse_server_policy,
+        args=args,
+        sub_object_id=name,
+        sub_object_key='name')
+    readable_output = tableToMarkdown(
+        name=OutputTitle.SERVER_POLICY_LIST.value,
+        metadata=pagination_message,
+        t=parsed_data,
+        headers=['name', 'deployment_mode', 'virtual_server', 'protocol', 'web_protection_profile', 'monitor_mode'],
+        headerTransform=string_to_table_header)
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='FortiwebVM.ServerPolicy',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def validate_custom_whitelist(version: str, args: Dict[str, Any], member_type: Optional[str] = None):
```

Comment:
this should be a method of the client. Put the common things in the base class, and the version-specific ones in the version classes.

Reply:
I would like to speak with you about that.